trojsten/polls/templates/trojsten/polls/view_question.html

```diff
@@ -0,0 +1,52 @@
+{% extends "trojsten/layout/main.html" %}
+{% load polls_parts %}
+
+{% block title %}Anketa{% endblock %}
+
+{% block page_header %}
+
+<h1> Anketa </h1>
+
+{% endblock %}
+
+{% block page_content %}
+
+{% if not questions %}
+
+    Neexistujú žiadne ankety.
+
+{% else %}
+
+    <h1> {{ current.text }} </h1>
+
+    {% show_time current %}
+
+    <form method="POST">
+    {% csrf_token %}
+    
+    <table>
+    {% for voteanswer in answers %}
+        <tr>
+            <td margin="10px" style="padding:0 10px 0 10px">{% if voteanswer.2 == user_vote %}<b>{% endif %} {{ voteanswer.0 }} {% if voteanswer.2 == user_vote %}</b>{% endif %}</td>

```

Comment:
Tieto styly je asi lepsie napchat do CSS.

Reply:
Ako mam to CSS prepojit s htmlkom? Potreboval by som do <head> dostat <link...>, ale do tej hlavicky asi neviem zasahovat z template-u. Alebo mam editovat globalne CSS subory (/static/css/ksp/ksp.min.css)? Len tie ma kazdy seminar svoje.

Reply:
Do templatu sa da zasahovat pomocou `{% addtoblock "css" %}`
Ale tieto veci zvykneme davat do globalneho css, nech memame milion malych css ktore sa loaduju nekonecno casu.

Tie veci patria sem:
https://github.com/trojsten/web/blob/master/css/src/base.less

potom si zbuildujes cssko podla navodu na https://github.com/trojsten/web/wiki/Kompilacia-CSS

Reply:
Dakujem. Preco je treba CSS kompilovat? Co to znamena?

Reply:
Nejak mi to nefunguje. Kompilacia vyzera uspesne, ale vo vyslednom css to, co som do base.less pridal, nie je.

Reply:
Pouzivame less, co je nadstavba nad css s podporou premennych a funkcii. Pouzivame bootstrap css, do ktoreho pridavame nejake vlastne veci a z celeho toho sa vygeneruje css pre kazdy seminar.

Zmeny by sa  mali prejavit v suboroch ako `trojsten/static/css/ksp/ksp.min.css` (git by mal vediet povedat ze su modifikovane).

Browser moze tie css cachovat, takze sa nemusia hned prejavit na stranke. Jednou z moznosti ako to okaslat je otvorit si dev consolu a zapnut "disable cache". Mozno by zafungoval aj refresh.

![Screenshot from 2020-01-01 12-07-58](https://user-images.githubusercontent.com/1177655/71641212-c416de80-2c8f-11ea-83d5-27631d42482c.png)